packages/user/src/model/invitations/service.ts

```diff
@@ -0,0 +1,42 @@
+import { BaseService } from "@dzangolab/fastify-slonik";
+
+import InvitationSqlFactory from "./sqlFactory";
+
+import type { Service } from "@dzangolab/fastify-slonik";
+import type { QueryResultRow } from "slonik";
+
+class InvitationService<
+    Invitation extends QueryResultRow,
+    InvitationCreateInput extends QueryResultRow,
+    InvitationUpdateInput extends QueryResultRow
+  >
+  extends BaseService<Invitation, InvitationCreateInput, InvitationUpdateInput>
+  // eslint-disable-next-line prettier/prettier
+  implements Service<Invitation, InvitationCreateInput, InvitationUpdateInput> {
+
+  get table() {
+    return "invitations";
```

Comment:
this is not how we define table name, it should be define in static TABLE property please check things service.